webview/__mocks__/@vscode/webview-ui-toolkit/react.tsx

```diff
@@ -28,3 +28,11 @@ export const VSCodeCheckbox: React.FC<MockCheckboxProps> = ({ onClick }) => {
 export const VSCodeProgressRing: React.FC = () => {
   return <div />
 }
+
+const MockComponentWithChildren: React.FC<PropsWithChildren> = ({
+  children
+}) => <div>{children}</div>
+
+export const VSCodePanels = MockComponentWithChildren
+export const VSCodePanelTab = MockComponentWithChildren
+export const VSCodePanelView = MockComponentWithChildren
```

Comment:
[F] Once we mock these components we no longer need `jest.mock('../../shared/components/codeSlider/CodeSlider')` in the setup App tests.